apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.tsx

```diff
@@ -77,6 +77,9 @@ function transformPoolCandidateSearchInputToFormValues(
       input?.applicantFilter?.expectedClassifications
         ?.filter(notEmpty)
         .map((c) => `${c.group}-${c.level}`) ?? [],
+    stream: input?.applicantFilter?.stream
+      ? [input?.applicantFilter?.stream]
+      : [],
```

Comment:
the stream from the request does not get loaded in the filters dialog initial values, it should
nor does it seem to do any filtering (no API call is made)

https://recordit.co/oEiD9kzSdA